packages/terra-clinical-onset-picker/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,25 @@
 # Terra Clinical Onset Picker Upgrade Guide
 
+## Changes from 3.0 to 4.0
+
+### Breaking Change
+* Condensed onChange function props into a single prop (`onsetOnChange`) that returns an Object for the onsetDate.
+
+## New Feature
+
+* Added support for age units.
+
+### Removed Props
+
+* Removed prop `granularitySelectOnChange`
+* Removed prop `precisionSelectOnChange`
+* Removed prop `onsetDateInputOnChange`
```

Comment:
Shouldn't this be listed under the Breaking change with instructures on how to convert to use onsetOnChange?

Reply:
Moved to breaking 77f1bc1